jupyter-notebooks/analytics/user-guide/2. Analytic Feeds Results.ipynb

```diff
@@ -68,14 +84,18 @@
     "| Vessel Detection | Object Detection | Detection Features (Polygons) | WFS 3 (Detection Features) |\n",
     "| Building Detection | Segmentation (Classification) | Raster Mask / Basemaps | WFS 3 (Quads) / XYZ Tiles |\n",
     "| Road Detection | Segmentation (Classification) | Raster Mask / Basemaps | WFS 3 (Quads) / XYZ Tiles |\n",
+    "| Building Construction Detection | Segmentation (Classification) | Raster Mask / Basemaps | WFS 3 (Quads) / XYZ Tiles |\n",
+    "| Road Construction Detection | Segmentation (Classification) | Raster Mask / Basemaps | WFS 3 (Quads) / XYZ Tiles |\n",
```

Comment:
shouldn't the result type be `Change Detection` not `Segmentation (Classification)`?

Reply:
Yes, I'll fix that.

Reply:
I've updated the table a bit.